cli/medperf/comms/rest.py

```diff
@@ -183,8 +183,10 @@ def get_benchmark_models(self, benchmark_uid: int) -> List[int]:
         Returns:
             list[int]: List of model UIDS
         """
-        models = self.__get_list(f"{self.server_url}/benchmarks/{benchmark_uid}/models")
-        model_uids = [model["id"] for model in models]
+        # TODO: filter to find approved ones only if we decide to have
```

Comment:
Should we handle filtering within the comms layer? Or within the entity layer? I think we've done filtering on the entity layer so far, because we also need to apply filtering to local entities

Reply:
It's tricky for associations since we don't have an association `Entity` on the client yet.

Filtering associations requires two steps:
1. first, finding the latest association of each mlcube. Ideally this should live in an association `Entity`, but for now I think this should live in the comms layer, since the client doesn't actually care about the history of associations. Also, this logic is required for all places in the client code where this comms function is called.
2. Then, ordinary filtering by approval status. Currently this filtering is required in two places:
  a. `entity.list` command: which already applies filtering in the commands layer.
  b. `benchmark.get_models_uids`: which by definition should return approved associations model IDs. We can apply filtering inside this function.

I think this will become cleaner if we decide to create an association `Entity` in our code base.